engine/apps/alerts/models/alert_group.py

```diff
@@ -899,7 +899,7 @@ def wipe_by_user(self, user: User) -> None:
             self.resolve(resolved_by=AlertGroup.WIPED)
             self.stop_escalation()
             self.distinction = ""
-            self.verbose_name = "Wiped incident"
+            self.verbose_name = None
```

Comment:
By this PR you change the purpose of `verbose_name` field of AlertGroup. It used to store rendered name for UI, now it acts more as an `search_cache`. It worth mentioning next to the model field definition. Also worth mentioning why exactly we don't want to migrate.

P.S. rock-n-roll could be renaming the field to smth self-explanory, but anchoring using `db_column='verbose_name'` to avoid migrations.

Reply:
👍 , I think we can just rename, should be pretty cheap in terms of migration